src/App.jsx

```diff
@@ -1,41 +1,58 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
+
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 
-const App = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App = () => {
+  const [selectedUserId, userSelect] = useState('');
+  const [selectedPostId, selectPostId] = useState('');
```

Comment:
```suggestion
  const [selectedUserId, setSelectedUserId] = useState('');
  const [selectedPostId, setSelectedPostId] = useState('');
```
Называй функцию так же, как и переменную, только с приставкой `set`